File: test-fixtures/cross-language/swift/realistic.swift
// SPDX-License-Identifier: MIT
public final class SessionState {
    public var mutableField: String = ""
    public let immutableField: String = "fixed"
    public static var staticField: Int = 0
    private var privateField: Bool = false
    public var sharedName: Int = 0
}

public final class AuditState {
    public var sharedName: Int = 0
}

File: test-fixtures/cross-language/java/realistic.java
// SPDX-License-Identifier: MIT
package crosslanguage.realistic;

public class SessionState {
    public String mutableField;
    public final String immutableField = "fixed";
    public static int staticField;
    private boolean privateField;
    public int sharedName;
}

class AuditState {
    public int sharedName;
}
